config/initializers/devise.rb

```diff
@@ -18,7 +18,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = ENV["MAIL_SENDER"]
```

Comment:
This is perfect! Now we have the email configured in an env variable, what do you think about moving it to a secret?
`config/secrets.yml`
```yaml
development:
  devise_mailer: <%= ENV.fetch('MAIL_SENDER', 'some_email@wolox.com.ar') %>

production: 
    devise_mailer: <%= ENV['MAIL_SENDER'] %>
```

As you see, we are using a default for development if there isn't any env variable set, but we are not using any default for production environment!